src/main/java/uk/nhs/digital/nhsconnect/nhais/outbound/OperationOutcomeExceptionHandler.java

```diff
@@ -42,8 +44,15 @@ public final ResponseEntity<String> handleAllErrors(Exception ex, WebRequest req
         return new ResponseEntity<>(content, headers, HttpStatus.INTERNAL_SERVER_ERROR);
     }
 
+
     @Override
-    protected ResponseEntity<Object> handleExceptionInternal(Exception ex, Object body, HttpHeaders headers, HttpStatus status, WebRequest request) {
+    protected ResponseEntity<Object> handleExceptionInternal(
+        Exception ex,
+        @Nullable Object body,
+        HttpHeaders headers,
+        HttpStatusCode status,
```

Comment:
This has been changed use the interface `HttpStatusCode` instead of `HttpStatus`